### PR TITLE
Add missing wrappers & fix README.md

### PR DESCRIPTION
**Pull request by e5l**

### CODE REVIEW

File: chat/README.md [hunk 1]
```diff
@@ -4,10 +4,10 @@ A simple chat application written with [Ktor](https://ktor.io) using websockets
 
 ## Running
 
-Execute this command in the repository's root directory to run this sample:
+Execute this command to run the sample:
 
 ```bash
-./gradlew :chat:run
+./gradlew :run
```
Comment (Stexxe) [inline chat/README.md:10]: Extra colon

File: client-tools/README.md [hunk 2]
```diff
@@ -4,10 +4,10 @@ Sample project for [Ktor](https://ktor.io) showing several useful extension meth
 
 ## Running
 
-Execute this command in the repository's root directory to run this sample:
+Execute this command to run the sample:
 
 ```bash
-./gradlew :client-tools:run
+./gradlew crun
```
Comment (Stexxe) [inline client-tools/README.md:10]: What is `crun`?

File: client-tools/gradle/wrapper/gradle-wrapper.properties [hunk 3]
```diff
@@ -1,5 +1,5 @@
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-7.1-bin.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-7.3-bin.zip
```
Comment (Stexxe) [inline client-tools/gradle/wrapper/gradle-wrapper.properties:3]: Is this PR about bumping the version of Gradle?

**Review by Stexxe — CHANGES_REQUESTED** [review]

File: chat/README.md (hunk 1 above)
Comment (e5l) [reply chat/README.md:10]: Thank you, fixed

File: client-tools/gradle/wrapper/gradle-wrapper.properties (hunk 3 above)
Comment (e5l) [reply client-tools/gradle/wrapper/gradle-wrapper.properties:3]: Nope, but I'm not sure why it should be an issue in this case

**Comment by e5l** [comment]
Fixed, please take a look

**Review by Stexxe — APPROVED** [review]

**Pull request merged**